### ISSUE TITLE
Incorrect typings custom_fields when 

### ISSUE THREAD

**Issue by leeuwd** [stripe/stripe-node#794]
When specifying [custom_fields](https://stripe.com/docs/api/customers/create#create_customer-invoice_settings-custom_fields) in the creation of a customer, the `InvoiceSettings` (line 282 in `types/2019-12-03/Customers.d.ts`) typings are incorrect.

When creating a customer adhering to these *incorrect* typings, the Stripe dashboard logs show:

![image](https://user-images.githubusercontent.com/1755190/73849534-634c6a00-482a-11ea-961f-a6c9e1841c7e.png)
 
The code used to POST (and adhering to current typings) was:

```js
const customer: Stripe.Customer = await this.client.customers.create({
      /* eslint-disable @typescript-eslint/camelcase */
      name: data.name,
      email: data.email,
      address: data.address,
      tax_id_data: [
        {
          type: 'eu_vat',
          value: data.vatId,
        },
      ],
      invoice_settings: {
        custom_fields: {
          name: 'KVK-nummer',
          value: data.cocNumber,
        },
      },
      /* eslint-enable @typescript-eslint/camelcase */
    })
```

It seems  `custom_fields` should be `InvoiceSettings.CustomFields[] | null`, *not* `InvoiceSettings.CustomFields | null`.

Library version: `stripe@8.14.0`.

**Comment by leeuwd** [comment]
Just confirmed it's working when sending and `InvoiceSettings.CustomFields[]` and suppressing Typescript warnings with `// @ts-ignore`.

**Comment by remi-stripe** [comment]
@leeuwd Thanks for the report! There's a PR opened to fix this here: https://github.com/stripe/stripe-node/pull/766
We can't merge it just yet as the types are automatically generated internally and we first need to fix our tooling to surface the right types but we're working on it

**Comment by richardm-stripe** [comment]
This is fixed as of #802. Thanks for the report!
